packages/react-sdk/src/components/link.tsx

```diff
@@ -1,9 +1,15 @@
 import { forwardRef, type ElementRef, type ComponentProps } from "react";
 
-type Props = Omit<ComponentProps<"a">, "href"> & { href?: string };
+type Props = Omit<ComponentProps<"a">, "href" | "target"> & {
+  href?: string;
+  target?: "self" | "blank" | "parent" | "top";
+  prefetch?: "none" | "intent" | "render";
```

Comment:
It's a bit wrong to have preload here since this is not an attribute of regular <a> link. Strictly speaking this prop should be defined in remix-link.tsx, and we should generate separate props for it etc.

But maybe we can be loose here for simplicity, I'd just add a code comment at least.

Reply:
yes and the way we have it here right now doesn't allow generating the props from the actual component being used, system is broken